frontend/editor.js

```diff
@@ -153,7 +153,7 @@ class PlutoEditorComponent extends HTMLElement {
     }
 
     connectedCallback() {
-        const new_launch_params = Object.fromEntries(Object.entries(launch_params).map(([k, v]) => [k, from_attribute(this, k) ?? v]))
+        const new_launch_params = Object.fromEntries(Object.entries(launch_params).map(([k, v]) => [k, from_attribute(this, k) || v]))
```

Comment:
Classic example of JS weirdness. Consider the `truthy()` value (true or false from def of `truthy` fn) that `disable_ui` returns when `from_attribute` is called. The `??` operator will return the first non-nullish value, which is guarenteed to be whatever truthy returns in `from_attribute`, be it true or false (since true and false are both non-nullish). This means that if `disable_ui` is set in `launch_params` but not in attributes, the `launch_params` value of `disable_ui` will still be overridden. This meant in static exports the UI was still being shown.

Switching to the `||` operator will yield the first truthy value (or the rightmost value if there are no truthy values) instead of the first nullish value, which I believe is sufficient to maintain the same behavior for other parameters while also properly handling boolean parameters. Let me know if I'm missing a case here though.

Reply:
Also found this while fixing this PR! Fixed in https://github.com/fonsp/Pluto.jl/commit/d2ea880ec38141f9064f09b446600dce2c4cbbbf